pom.xml

```diff
@@ -19,9 +19,9 @@
 
   <properties>
     <!-- Dependency versions -->
-    <commonreportsVersion>1.2.0</commonreportsVersion>
+    <commonreportsVersion>1.3.0-SNAPSHOT</commonreportsVersion>
     <odoo_initializerVersion>2.1.0</odoo_initializerVersion>
```

Comment:
What about this one then?

Reply:
I could change that one as well. I wasn't quite aware how much of a moving part it is.

Reply:
Not much has changed  on `odoo_initializer` though.